src/models/sqlOutputContentProvider.ts

```diff
@@ -298,7 +299,7 @@ export class SqlOutputContentProvider {
         const queryRunner = this.getQueryRunner(uri);
         // in case of a tab switch
         // and if it has rendered before
-        if (panelController.isActive &&
+        if (panelController.isActive !== undefined &&
```

Comment:
What is this fixing? Seems odd to be fine with both true and false as values here.

Reply:
This is telling the webview to refresh if `panelController.isActive` isn't undefined, then it's being refreshed. 
This is fixing the scenario when let's say you have 2 editor tabs open - Untilted-1 and Untitled-2, with results for Untilted-2 open on the side. Then, running a query from Untitled-1 should bring the correct corresponding results to foreground, while maintaining focus on the editor.